lib/utils/index.js

```diff
@@ -533,6 +533,29 @@ class Utils {
   static shouldReplaceStack(err) {
     return !alwaysDisplayError(err);
   }
+
+  static findTSConfigFile() {
+    const projectTsFileLocation1 = path.join(process.cwd(), 'nightwatch', 'tsconfig.json');
+    const projectTsFileLocation2 = path.join(process.cwd(), 'tsconfig.nightwatch.json');
+    if (Utils.fileExistsSync(projectTsFileLocation1)) {
```

Comment:
We can actually add a new setting called `tsconfig_path` (empty by default) so users can set that to whatever they want, in which case they can re-use the existing tsconfig file if they want to. If this property is set, then it should take precedence.

Reply:
@vaibhavsingh97 any progress with adding the `tsconfig_path` setting?